src/app/frontend/replicaset/detail/info_component.js

```diff
@@ -34,6 +38,13 @@ export default class ReplicaSetInfoController {
   areDesiredPodsRunning() {
     return this.replicaSet.podInfo.running === this.replicaSet.podInfo.desired;
   }
+
+  getLogsHref() {
+    return this.state_.href(
+        logsStateName,
+        new LogsStateParams(
+            this.replicaSet.objectMeta.namespace, this.replicaSet.objectMeta.name, 'ReplicaSet'));
```

Comment:
We have no specific pattern. If you use `'ReplicaSet'` more than once within a file it would be good to export it to `const` to ensure, that there are no typos etc.